apps/perf-test/src/scenarios/FocusZone.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { FocusZone } from 'office-ui-fabric-react';
+
+const FocusZoneExample: React.FunctionComponent = () => {
+  const items: React.ReactNode[] = [];
+
+  for (let i = 0; i < 20; i++) {
+    items.push(<li key={i} data-is-focusable={true} />);
+  }
```

Comment:
>This will run for every instance of example rendered. Is that what you intend?
this is rendering the child list items, so yes this is intended.

I did minor clean up so the `FocusZoneContent` shows up in the flame chart

Reply:
Curious why you would want render time for children to appear? I don't think `FocusZone` will impact how long it takes to initially render its children. It seems it would actually obfuscate rendering issues inside `FocusZone` itself.